main/pgpool/APKBUILD

```diff
@@ -24,12 +27,15 @@ prepare() {
 build() {
 	cd "$builddir"
 	./configure \
-		--build=$CBUILD \
-		--host=$CHOST \
+		--build="$CBUILD" \
+		--host="$CHOST" \
 		--prefix=/usr \
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
-		--infodir=/usr/share/info
+		--infodir=/usr/share/info \
+		--with-memcached=/usr \
+		--with-openssl \
+		--with-pam
```

Comment:
Why do we need PAM support? Do we really want that? Can you please include a good reason to why we should add PAM support in the commit message if you still think this is a good idea.

We normally try avoid PAM if possible, and it may be difficult to remove it once its added, so we just need to be sure that this is really needed.

Reply:
Removed `pam` from configure options.